Gemfile.lock

```diff
@@ -451,6 +451,3 @@ DEPENDENCIES
   webmock
   whenever
   will_paginate
-
-BUNDLED WITH
-   1.10.6
```

Comment:
If your bundler removes it, it means it's an old version. `gem install bundler` helps.